blog/features/replica-signed-queries.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Announcing replica signed queries"
+description: To enhance the trustworthiness of queries by providing better guarantees against attacks, we're excited to announce the release of replica signed queries!
+tags: [New features]
+image: /img/blog/replica-signed-queries.jpg
```

Comment:
Great. I have seen the webpage after the PR is merged.  https://internetcomputer.org/blog/features/replica-signed-queries
However, the image does  not show up. Is it because the image path here does not match the image name
`dev-blog-replica-signed-queries.jpg`?